neuro_extras/main.py

```diff
@@ -1000,10 +1000,18 @@ async def _run_copy_container(src_cluster: str, src_path: str, dst_path: str) ->
     ]
     cmd = " ".join(args)
     print(f"Executing '{cmd}'")
-    subprocess = await asyncio.create_subprocess_shell(cmd)
+    subprocess = await asyncio.create_subprocess_shell(
+        cmd, stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE
+    )
+    stdout, stderr = await subprocess.communicate()
     returncode = await subprocess.wait()
     if returncode != 0:
-        raise Exception("Unable to copy storage")
+        raise Exception(
+            "Unable to copy storage\n"
+            + stdout.decode("utf-8")
+            + "\n"
+            + stderr.decode("utf-8")
+        )
```

Comment:
1. to avoid `stdout.decode`, just run subprocess with `encoding='utf-8'`
2. this will duplicate error in the real user's console (bc the user sees all teh stdout and stderr live). Let's revert this change and better [patch tests](https://github.com/neuro-inc/neuro-extras/pull/113)?